microsoft-365/enterprise/manage-microsoft-365-groups-with-powershell.md

```diff
@@ -104,32 +104,32 @@ Check out [Using PowerShell with Exchange Online](https://docs.microsoft.com/pow
 
 Once these settings are enabled, the group owner will be able to choose a classification from the drop down menu in Outlook on the Web and Outlook, and save it from the **Edit** group page.
 
-![Choose Office 365 group classification](../media/f8d4219a-6180-491d-b0e1-4313ac83998b.png)
+![Choose MIcrosoft 365 Group classification](../media/f8d4219a-6180-491d-b0e1-4313ac83998b.png)
 
-## Hide Office 365 Groups from GAL
+## Hide MIcrosoft 365 Groups from GAL
 <a name="BKMK_CreateClassification"> </a>
 
-You can specify whether an Office 365 group appears in the global address list (GAL) and other lists in your organization. For example, if you have a legal department group that you don't want to show up in the address list, you can stop that group from appearing in GAL. Run the Set-Unified Group cmdlet to hide the group from address list like this:
+You can specify whether an MIcrosoft 365 Group appears in the global address list (GAL) and other lists in your organization. For example, if you have a legal department group that you don't want to show up in the address list, you can stop that group from appearing in GAL. Run the Set-Unified Group cmdlet to hide the group from address list like this:
 
 ```powershell
 Set-UnifiedGroup -Identity "Legal Department" -HiddenFromAddressListsEnabled $true
 ```
 
-## Allow only internal users to send message to Office 365 group
+## Allow only internal users to send message to MIcrosoft 365 Groups
 <a name="BKMK_CreateClassification"> </a>
 
-If you don't want users from other organization to send email to an Office 365 group, you can change the settings for that group. It will allow only internal users to send an email to your group. If external user try to send message to that group they will be rejected.
+If you don't want users from other organization to send email to an MIcrosoft 365 Group, you can change the settings for that group. It will allow only internal users to send an email to your group. If external user try to send message to that group they will be rejected.
```

Comment:
```suggestion
If you don't want users from other organizations to send emails to a Microsoft 365 Group, you can change the settings for that group. It will allow only internal users to send an email to your group. If an external user tries to send a message to that group, it will be rejected.
```